RobotStateMessage sourceRobotState
---
geometry_msgs/Pose cameraFrame